### DISCUSSION TITLE
No proper documentation for _document.tsx......

### DISCUSSION THREAD

**Discussion by athulnanda123** [vercel/next.js#30121] [Help]
struggling with typescript, please add a perfect documentation on  how to properly configure _document.tsx, only _document.js documentation is complete in nexjs official documentation.

**Comment by jamesmosier** [comment]
The docs have a section for using Typescript within your custom document:

https://nextjs.org/docs/advanced-features/custom-document#typescript

**Reply by athulnanda123** [reply]
Official Documentation is too short to understand, it don't have proper configuration instructions, can you give an elaborated example reference ?
